cypress/integration/basic/person_modal.spec.js

```diff
@@ -0,0 +1,28 @@
+describe('Test that the person show modal can be opened', () => {
+  const BOOKMARKED_PERSON = 'Sadaf Arshad';
+
+  it('Bookmark a person', () => {
+    cy.visit('/applications');
+
+    cy.get('input#search_sidebar.input-large.search-query')
+    .type(BOOKMARKED_PERSON, {delay: 100})
+    .should('have.value', BOOKMARKED_PERSON)
+    .type('{enter}');
+  });
+
+  it('Clicking a bookmarked person\'s open icon should reveal a modal',() => {
+    cy.get('div#sidebar-area a.entity-details-link:first').click({force: true});
+  });
+
+  it('The modal should have the person\'s name.', () => {
+    cy.get('div.modal-header h3').should('contain', BOOKMARKED_PERSON);
```

Comment:
This test broke on my end because I had different bookmarks in my database.

Is it possible to store variables between it()s? If so, can the test instead ensure that the h3 contains whatever name was indicated by the right sidebar pin that was clicked on, instead of assuming a specific name?